db/revision_cache_interface.go

```diff
@@ -337,10 +337,10 @@ func revCacheLoaderForDocumentCV(ctx context.Context, backingStore RevisionCache
 	if err = doc.HasCurrentVersion(cv); err != nil {
 		return bodyBytes, body, history, channels, removed, attachments, deleted, doc.Expiry, revid, err
 	}
-	channels = doc.currentRevChannels
+	doc.currentRevChannels = doc.SyncData.getCurrentChannels()
 	revid = doc.CurrentRev
 
-	return bodyBytes, body, history, channels, removed, attachments, deleted, doc.Expiry, revid, err
+	return bodyBytes, body, history, doc.currentRevChannels, removed, attachments, deleted, doc.Expiry, revid, err
```

Comment:
Is there anywhere in the code we're reading the doc.currentRevChannels property other than this spot?  If so, and we're recomputing it here anyway, we should just remove the currentRevChannels property.

Reply:
This was one part that was wrapped up in the revtree channel changes in #6368 but a version of it still ended up making its way into beryllium via another PR (#6491)

I'll strip this PR back more and just use `getCurrentChannels()` without setting the field until it's necessary.